ci/authn-gcp/get_gcp_id_tokens.sh

```diff
@@ -0,0 +1,154 @@
+#!/bin/bash
+
+#	---------------------------------------------------------------------------------------------------------------------
+# This script obtains all of the GCP identity tokens required by authn-gcp cucumber tests
+# The scripts accepts the GCP instance name as an argument.
+# The script does the following:
+# 1. Validates gcloud is installed (requires a GCP account)
+#   To install gcloud goto: https://cloud.google.com/sdk/docs, download and install the SDK by running:
+#   ./google-cloud-sdk/install.sh
+#   ./google-cloud-sdk/bin/gcloud init
+# 2. Verifies the GCP instance exists and is running
+# 3. Executes a ssh curl command and write the output token with the appropriate token name to '../ci/authn-gcp/tokens'.
+#	---------------------------------------------------------------------------------------------------------------------
+
+PROGNAME=$(basename "$0")
+INSTANCE_ZONE=""
+TOKENS_OUT_DIR_PATH=../ci/authn-gcp/tokens
+INSTANCE_EXISTS=0
+INSTANCE_RUNNING=0
+
+main() {
+  echo "-- ------------------------------------------ --"
+  echo "-- Generate Google Cloud GCP Identity tokens --"
+  echo "-- ------------------------------------------ --"
+
+  echo "-- Verifying 'gcloud' is installed..."
+  ensure_gcloud_is_installed
+  echo "-- Verifying GCP instance name..."
+  check_instance_name_arg "$1"
+  echo "-- Checking if GCP instance: '${INSTANCE_NAME}' exists..."
+  ensure_instance_exists_and_running "$INSTANCE_NAME"
+  echo "-- Generate tokens and writing to files under '${TOKENS_OUT_DIR_PATH}'..."
```

Comment:
I think we should add a step before which validate/delete the output dir, to prevent an errors of using an old tokens from previous runs